packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2775,6 +2825,35 @@ export class StripeHelper extends StripeHelperBase {
       );
     }
 
+    let remainingAmountTotal: number | undefined;
+    let unusedAmountTotal = 0;
+
+    if (invoice.lines.data) {
+      const totals = invoice.lines.data.reduce(
+        (totals, line) => {
+          if (line.proration === true) {
+            const amount = line.amount || 0;
+            const description = line.description || '';
+
+            if (amount < 0 && /^Unused/i.test(description)) {
+              totals.unusedAmountTotal += amount;
+            } else if (amount > 0 && /^Remaining/i.test(description)) {
+              totals.remainingAmountTotal =
+                (totals.remainingAmountTotal ?? 0) + amount;
+            }
+          }
+          return totals;
+        },
+        {
+          remainingAmountTotal: undefined as number | undefined,
```

Comment:
Does `remainingAmountTotal` need to be `undefined` rather than `0` downstream?

Reply:
I'm not totally familiar with the email template syntax, but it looks like this is just checked against its truthyness value, so 0 and `undefined` should equate to the same thing